### DISCUSSION TITLE
So... I unbound the alt scroll to move selections

### DISCUSSION THREAD

**Discussion by mikeyboy** [maruohon/litematica#366] [Q&A]
I was wondering what the name of the keybind is and how to get it back? Thanks.

**Comment by maruohon** [comment]
Did you just unbind everything? Why? Did you have some conflicts with other things?

Anyway, that one is called `selectionNudgeModifier`.

**Reply by mikeyboy** [reply]
uhm yeah but i think everything is back to normal (hopefully) 😄
